_wiki/Config.md

```diff
@@ -17,6 +17,8 @@ The script acts depending on a set of variables. In order to change these, open
 | :-------------------- | :-------: | :-------------------------------------------------------------------------------------------------------------------------------------------------------- | :-----: |
 | `canOpenSoren`        | `Boolean` | If `true`, player has permission to open Soren.                                                                                                           | `false` |
 | `arenaHeroesOpponent` | `Number`  | Choose which opponent to fight in the Arena of Heroes. Possible entries: `1`, `2`, `3`, `4` or `5`, where `1` is the top opponent and `5` the bottom one. |   `5`   |
+| `vipLevel`            | `Number`  | Current VIP level of the account.                                                                                                                         |   `0`   |
+| `campaignStage`       | `Number`  | Current campaign stage of the account.                                                                                                                    |   `0`   |
```

Comment:
Not a big fan of making this a number, as stages always come in pairs: first the world, then the stage. And they are important, as some things unlock in the middle of a world, for example at [`12-40`](https://github.com/zebscripts/AFK-Daily/wiki/AFK-Arena-requirements).

I tried to fix this myself, but ran into a few problems: For example running `./deploy.sh` once would add `campaignStage="0-0"`, but running it again would remove the `"`, making it an integer expression.

I do believe it's important to get this right, as it will save us headaches in the future.

Reply:
Rah, it's boring to check if we do it that way, maybe remove the dash? 👿

Reply:
Hm, alright, I guess that is possible, yes. Would save us a headache for now I guess, right? As long as `string`s don't contain spaces in them it's fine, though I'm sure we'll have to change this one day.

Reply:
Note for me
```console
$ a="3-3"; [ ${a//-} -eq 33 ] && echo "ok"
ok
```